code/modules/unit_tests/examine.dm

```diff
@@ -0,0 +1,37 @@
+// Tests masks/hardsuits not properly hiding gender
+
+// To hide the gender the carbon has to be covering his body AND his face
+
+/datum/unit_test/examine/start()
+
+/datum/unit_test/examine/hidden_identity/start()
+	var/mob/living/carbon/human/test_subject = new()
+	var/name = "Barack Obama"
+	test_subject.real_name = name
+	test_subject.name = name
```

Comment:
Given that you go on to test the pronouns later, you should probably explicitly specify gender somewhere in here